enterprise/backend/test/metabase_enterprise/advanced_permissions/models/permissions/block_permissions_test.clj

```diff
@@ -113,7 +113,9 @@
               (is (= {:schemas :block}
                      (test-db-perms group-id)))
               (is (= #{(perms/database-block-perms-path (mt/id))}
-                     (db/select-field :object Permissions :group_id group-id))))))))))
+                     (db/select-field :object Permissions {:where [:and
+                                                                   [:<> :object (perms/general-perms-path :subscription)]
```

Comment:
Probably better to just check that `object LIKE '/block/%'` or something like that instead so we don't have to keep adding more `object <>` clauses in the future when we add more general perms types